src/ui/templates/insurance/account/create/your-details.njk

```diff
@@ -0,0 +1,151 @@
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: validationErrors.summary
+    }) }}
+  {% endif %}
+
+  <h1 class="govuk-heading-xl" data-cy="heading">{{ CONTENT_STRINGS.PAGE_TITLE }}</h1>
+
+  <p data-cy="intro">{{ CONTENT_STRINGS.INTRO }}</p>
+
+  <form method="POST" novalidate>
+
+    <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-one-half-from-desktop">
+
+        {{ govukInput({
+          label: {
+            text: FIELDS.FIRST_NAME.LABEL,
+            classes: "govuk-!-font-weight-bold govuk-!-font-size-24",
+            attributes: {
+              id: FIELDS.FIRST_NAME.ID + '-label',
+              'data-cy': FIELDS.FIRST_NAME.ID + '-label'
+            }
+          },
+          attributes: {
+            'data-cy': FIELDS.FIRST_NAME.ID + '-input'
+          },
+          id: FIELDS.FIRST_NAME.ID,
+          name: FIELDS.FIRST_NAME.ID,
+          value: submittedValues[FIELDS.FIRST_NAME.ID],
+          errorMessage: validationErrors.errorList[FIELDS.FIRST_NAME.ID] and {
+            text: validationErrors.errorList[FIELDS.FIRST_NAME.ID].text,
+            attributes: {
+              "data-cy": FIELDS.FIRST_NAME.ID + "-error-message"
+            }
+          }
+        }) }}
+
+        {{ govukInput({
+          label: {
+            text: FIELDS.LAST_NAME.LABEL,
+            classes: "govuk-!-font-weight-bold govuk-!-font-size-24",
+            attributes: {
+              id: FIELDS.LAST_NAME.ID + '-label',
+              'data-cy': FIELDS.LAST_NAME.ID + '-label'
+            }
+          },
+          attributes: {
+            'data-cy': FIELDS.LAST_NAME.ID + '-input'
+          },
+          id: FIELDS.LAST_NAME.ID,
+          name: FIELDS.LAST_NAME.ID,
+          value: submittedValues[FIELDS.LAST_NAME.ID],
+          errorMessage: validationErrors.errorList[FIELDS.LAST_NAME.ID] and {
+            text: validationErrors.errorList[FIELDS.LAST_NAME.ID].text,
+            attributes: {
+              "data-cy": FIELDS.LAST_NAME.ID + "-error-message"
+            }
+          }
+        }) }}
+
+        {{ govukInput({
+          label: {
+            text: FIELDS.EMAIL.LABEL,
+            classes: "govuk-!-font-weight-bold govuk-!-font-size-24",
+            attributes: {
+              id: FIELDS.EMAIL.ID + '-label',
+              'data-cy': FIELDS.EMAIL.ID + '-label'
+            }
+          },
+          attributes: {
+            'data-cy': FIELDS.EMAIL.ID + '-input'
+          },
+          id: FIELDS.EMAIL.ID,
+          name: FIELDS.EMAIL.ID,
+          value: submittedValues[FIELDS.EMAIL.ID],
+          errorMessage: validationErrors.errorList[FIELDS.EMAIL.ID] and {
+            text: validationErrors.errorList[FIELDS.EMAIL.ID].text,
+            attributes: {
+              "data-cy": FIELDS.EMAIL.ID + "-error-message"
+            }
+          }
+        }) }}
+
+        {{ govukInput({
```

Comment:
does this field show or hide the password?

Reply:
Not yet - this is in another ticket.

ATM, just an empty field that doesn't work or do anything, just visual display.